pkg/pex/pex.go

```diff
@@ -95,15 +96,16 @@ func New(ctx context.Context, h host.Host, opt ...Option) (px *PeerExchange, err
 
 func (px *PeerExchange) Loggable() map[string]interface{} {
 	return map[string]interface{}{
-		"id": px.h.ID(),
-		"view_size":  px.gossip.C,
+		"id":        px.h.ID(),
+		"view_size": px.gossip.C,
 	}
 }
 
 func (px *PeerExchange) Close() error { return px.runtime.Stop(context.Background()) }
 
 // Bootstrap a namespace by performing an initial gossip round with a known peer.
 func (px *PeerExchange) Bootstrap(ctx context.Context, ns string, peer peer.AddrInfo) error {
+
```

Comment:
Drop extra newline

Reply:
What do you mean, two newlines?

Reply:
Oh, no, I just meant that we should remove the extra newline between `Bootstrap` and the start of the function body.